docs/modules/policies/pages/conditions.adoc

```diff
@@ -29,6 +30,8 @@ Within a condition block you have access to several special variables:
 
 Every condition expression must evaluate to a boolean true/false value. You can combine complex expressions together in a single condition block by using the `all`, `any`, or `none` operators.
 
+TIP: If the expression starts with a quotation mark, then it needs to be moved to a new line via the `>` syntax in the example above.
```

Comment:
```suggestion
.Quotes in expressions
****
Because single and double quotes have special meanings in YAML, if your expression contains quotation marks, use the YAML literal block syntax to avoid YAML parsing errors.

[source,yaml]
----
expr: |
  "GB" in R.attr.geographies
----  
****
```